modules/ossm-security-mtls.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/service_mesh_user_guide/ossm-security.adoc
+
+[id="ossm-security-mtls_{context}"]
+= Enabling mutual Transport Layer Security (mTLS)
+
+Mutual Transport Layer Security (mTLS) is a protocol where two parties authenticate each other at the same time. It is the default mode of authentication in some protocols (IKE, SSH) and optional in others (TLS).
+
+MTLS can be used without changes to the application or service code. The TLS is handled entirely by the service mesh infrastructure and between the two sidecar proxies.
+
+By default, {ProductName} is set to permissive mode, where the sidecars in {ProductShortName} accept both plain-text traffic and connections that are encrypted using mTLS. If a service in your mesh is communicating with a service outside the mesh, strict mTLS could break communication between those services. Use permissive mode while you migrate your workloads to {ProductShortName}.
+
+== Enabling strict mTLS across the mesh
+
+If your workloads do not communicate with services outside your mesh and communication will not be interrupted by only accepting encrypted connections, you can enable mTLS across your mesh quickly. Set `.Spec.istio.global.mtls.enabled` to `true` in your ServiceMeshControlPlane resource. The operator create the required resources.
+
+[id="ossm-security-mtls-sidecars-incoming_{context}"]
+== Configuring sidecars for incoming connections
+
+Create a ServiceMeshPolicy to enforce mTLS encryption on incoming connections. It instructs all sidecars to reject plain text connections and accept mTLS connections. The resource must be named `"default"` and it must be created in a project in your control plane.
```

Comment:
- [...] and *only* accept mTls connections.`
- [...] and it must be created in the same project as your ServiceMeshControlPlane resource.